test/dynamo/test_comptime.py

```diff
@@ -40,8 +40,9 @@ def _(ctx):
         self.assertExpectedInline(
             FILE.getvalue().strip(),
             """\
-def forward(self, x : torch.Tensor):
-    mul = x * 2;  x = None""",
+def forward(self, L_x_ : torch.Tensor):
```

Comment:
This renaming is truly a travesty. We should figure out how to get the original local as the input name...

Reply:
(Doesn't have to be this PR)

Reply:
Lemme play with that, I can stack it

Reply:
Stacked.